client/connectivity/balancer.go

```diff
@@ -0,0 +1,50 @@
+package connectivity
+
+import (
+	pb "github.com/sgielen/rufs/proto"
+	"google.golang.org/grpc/balancer"
+	"google.golang.org/grpc/balancer/base"
+)
+
+const BalancerName string = "rufs"
```

Comment:
nit: "rufs-peer" or something? (Given that we don't use it for connecting to the discovery server)

Reply:
Ah yes, fixed